AlbumList/AlbumListViewController.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  AlbumListViewController.swift
+//  AlbumList
+//
+//  Created by tommy tayler on 28/01/23.
+//
+
+import UIKit
+
+var albumTitle = ""
+var albumId = 0
+var myIndex = 0
+
+class AlbumListViewController: UITableViewController, StoryboardInitializable {
+
+
+    var albums = [Album]()
+    var users = [User]()
+    
+    
+    
+    @IBOutlet weak var albumTableView: UITableView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        // on load call API
+        
+        APIAlbums {
+            self.albumTableView.reloadData()
+            print("Albums success")
+        }
+        APIUsers()
+    }
+    
+    // MARK: - Table View Data Source
+    override func tableView(
+      _ tableView: UITableView,
+      numberOfRowsInSection section: Int
+    ) -> Int {
+        return albums.count // [albumList].count
+    }
+
+    override func tableView(
+      _ tableView: UITableView,
+      cellForRowAt indexPath: IndexPath
+    ) -> UITableViewCell {
+        let cell =  UITableViewCell(style: .default, reuseIdentifier: nil)
+        let row = indexPath.row
+        let album = albums[row]
```

Comment:
With the viewmodel approach, the VM should return the album to the view controller (VC)